### ISSUE TITLE
In fact, your SR is 99% only in the case of untargeted attack

### ISSUE THREAD

**Issue by Sunshine352** [mathcbc/advGAN_pytorch#1]

**Comment by mathcbc** [comment]
the accuracy of the targeted model toward adversarial examples is 1%，so the attack success rate (SR) is 99%, I think my code is correct.




On 11/23/2018 13:03, Zhaomang Sun wrote:

pred_lab = torch.argmax(target_model(adv_img), 1)
num_correct += torch.sum(pred_lab==test_label, 0)
pred_lab should be equal to targeted label, which is different from ground-truth label

**Comment by Sunshine352** [comment]
Oh, it's right. But you test SR in the case of targeted attack?

**Comment by Sunshine352** [comment]
In addition,  the line 'num_correct += torch.sum(pred_lab==test_label,0)' , pred_lab is the label of adv-image, test_label is the gt-label. Though target_model acc is high enough, the test-acc in test set is not 100%. So you should conduct the adversarial attack in the correct-classification samples among test set.

**Comment by mathcbc** [comment]
No, it's untargeted attack.




On 11/23/2018 16:55, Zhaomang Sun wrote:

Oh, it's right. But you test SR in the case of targeted attack?
